addons/web/static/tests/widgets/data_export_tests.js

```diff
@@ -100,16 +94,73 @@ QUnit.module('widgets', {
             viewOptions: {
                 hasActionMenus: true,
             },
-            mockRPC: this.mockDataExportRPCs,
-            session: {
-                ...this.mockSession,
-                get_file: function (params) {
-                    assert.step(params.url);
-                    params.complete();
-                },
+            mockRPC: function (route, args) {
+                if (route === '/web/dataset/call_kw/ir.async/search_read') {
+                    return Promise.resolve([]);
+                }
```

Comment:
If you turn the mockRPC function async, you don't have to wrap every return in a `Promise.resolve`:
```js
async mockRPC(route, args) {
   if (route === /* ... */) {
      return [];
   }
   // ...
}
```
Would do this to all tests.

Reply:
What's about ?

    if (route === '/longpolling/poll') {
        self.pollPromise = testUtils.makeTestPromise();
        self.pollPromise.abort = () => {self.pollPromise.resolve([], $.Event())};
        return self.pollPromise;
    }